packages/theme-default/src/components/Search/logic/providers/LocalProvider.ts

```diff
@@ -63,61 +69,83 @@ export class LocalProvider implements Provider {
       headers: page.toc.map(header => normalizeTextCase(header.text)).join(' '),
       normalizedTitle: normalizeTextCase(page.title),
     }));
-    const createOptions: CreateOptions = {
-      tokenize: 'full',
-      async: true,
-      doc: {
-        id: 'routePath',
-        field: ['normalizedTitle', 'headers', 'normalizedContent'],
-      },
-      cache: 100,
-      split: /\W+/,
-    };
+    const createOptions: IndexOptionsForDocumentSearch<PageIndexInfo[], true> =
+      {
+        tokenize: 'full',
+        document: {
+          id: 'id',
+          store: true,
+          index: ['normalizedTitle', 'headers', 'normalizedContent'],
+        },
+        cache: 100,
+        // charset: {
+        //   split: /\W+/,
```

Comment:
It this config no longer required?

Reply:
They made a breaking change to this API; I'm not sure about the effect for now. So, I saved the configuration in a comment, maybe someday we will need to revert to it.

Reply:
get 👌